extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -52,11 +54,15 @@ export class AmpStoryRequestService {
    * @private
    */
   loadJsonFromAttribute_(attributeName) {
-    if (!this.storyElement_.hasAttribute(attributeName)) {
+    const bookendEl = childElementsByTag(this.storyElement_,
```

Comment:
Nit: ``childElementByTag(this.storyElement_, 'amp-story-bookend')`` like Alan suggested : ))

Reply:
Oops, didn't see that comment. Thanks!